package.json

```diff
@@ -115,7 +115,7 @@
     "@commitlint/travis-cli": "^8.3.5",
     "@hapi/hapi": "^18.4.1",
     "@koa/router": "^8.0.8",
-    "@types/node": "^13.7.4",
+    "@types/node": "8.5.x",
```

Comment:
```suggestion
    "@types/node": "~8.5.10",
```

Reply:
Ah forgot about the tidle. Much better 👍